app/views/shared/_alert.erb

```diff
@@ -2,7 +2,7 @@
   <div>
     <% if flash[:alert] %>
       <div class="alert alert-danger" data-testid="alert-error">
-        <% flash[:alert].each do |error| %>
+        <% flash_alert.each do |error| %>
```

Comment:
How about 
```suggestion
        <% flash_alerts.each do |error| %>
```

Reply:
Agree. Fixed in 8860815636c5b46d8a31abe547c5a901e574c3e9